core/src/main/java/tc/oc/pgm/stats/TeamInventoryMenuItem.java

```diff
@@ -0,0 +1,151 @@
+package tc.oc.pgm.stats;
+
+import static tc.oc.pgm.stats.StatsMatchModule.PlayerStats.statFormat;
+import static tc.oc.pgm.stats.StatsMatchModule.numberComponent;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.kyori.text.Component;
+import net.kyori.text.TranslatableComponent;
+import net.kyori.text.format.TextColor;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.inventory.ClickType;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.match.MatchScope;
+import tc.oc.pgm.api.party.Competitor;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.menu.InventoryMenu;
+import tc.oc.pgm.menu.InventoryMenuItem;
+import tc.oc.pgm.menu.InventoryMenuUtils;
+import tc.oc.pgm.util.bukkit.BukkitUtils;
+import tc.oc.pgm.util.text.TextTranslations;
+
+public class TeamInventoryMenuItem implements InventoryMenuItem {
+
+  private final Competitor team;
+  private final InventoryMenu teamSubGUI;
+
+  private final TextColor RESET = TextColor.WHITE;
+
+  TeamInventoryMenuItem(Match match, Competitor team) {
+    this.team = team;
+    this.teamSubGUI =
+        InventoryMenuUtils.prettyMenu(
+            match,
+            TranslatableComponent.of("match.stats.title"),
+            team.getPlayers().stream()
+                .map(PlayerStatsInventoryMenuItem::new)
+                .collect(Collectors.toList()));
+  }
+
+  @Override
+  public Component getName() {
+    return TranslatableComponent.of("match.stats.team", team.getName());
+  }
+
+  @Override
+  public ChatColor getColor() {
+    return ChatColor.valueOf(team.getColor().name());
+  }
+
+  @Override
+  public List<String> getLore(MatchPlayer player) {
+
+    StatsMatchModule smm = player.getMatch().needModule(StatsMatchModule.class);
+    List<String> lore = new ArrayList<>();
+    int teamKills = 0;
+    int teamDeaths = 0;
+    double damageDone = 0;
+    double damageTaken = 0;
+    double bowDamage = 0;
+    int shotsTaken = 0;
+    int shotsHit = 0;
+    for (MatchPlayer teamPlayer : team.getPlayers()) {
+      StatsMatchModule.PlayerStats stats = smm.getPlayerStat(teamPlayer.getId());
+      teamKills += stats.kills;
+      teamDeaths += stats.deaths;
+      damageDone += stats.getDamageDone();
+      damageTaken += stats.getDamageTaken();
+      bowDamage += stats.getBowDamage();
+      shotsTaken += stats.shotsTaken;
+      shotsHit += stats.shotsHit;
+    }
+
+    double teamKD = teamDeaths == 0 ? teamKills : statFormat(teamKills / (double) teamDeaths);
+    // You hit 100% of the shots you dont take? :thinking:
+    double teamBowAcc = shotsTaken == 0 ? 100 : statFormat(shotsHit / (shotsTaken / (double) 100));
```

Comment:
Consider leaving a `Double.NaN` in here, and on the component display as a `-` or similar, to show no number rather than 100% or 0%